### PR TITLE
Update mac circleci environment

### PR DESCRIPTION
**Pull request by sbc100**
Apparently `Image xcode:9.0 is not supported` these days.

### CODE REVIEW

**Comment by sbc100** [comment]
Landed now to green the CI

File: emsdk [hunk 1]
```diff
@@ -11,11 +11,17 @@ if [ -z "$EMSDK_PYTHON" ]; then
...
     # When using our bundled python we never want the users
     # PYTHONHOME or PYTHONPATH
     # https://github.com/emscripten-core/emsdk/issues/598
     unset PYTHONHOME
     unset PYTHONPATH
+
+    # This is needed for MacOS.  Without this, the urlopen
+    # code will try to use /usr/local/etc/openssl/cert.pem
+    # which may or may not exist on the system.
+    export SSL_CERT_FILE=$(dirname $0)/python/3.7.4-2_64bit/lib/python3.7/site-packages/certifi/cacert.pem
```
Comment (kripken) [inline emsdk:24]: Why is this needed in two places?

Comment (sbc100) [reply emsdk:24]: Its needed here so that emsdk works correctly even the sdk is not active in the current env.  This block of code here runs when EMSDK_PYTHON is *not* set... which means that sdk has not activated in the current env.
